platform/osx/os_osx.mm

```diff
@@ -1237,7 +1237,7 @@ static void displays_arrangement_changed(CGDirectDisplayID display_id, CGDisplay
 	ERR_FAIL_COND_V(window_object == nil, ERR_UNAVAILABLE);
 
 	window_view = [[GodotContentView alloc] init];
-	if (@available(macOS 10.14, *)) {
+	if (floor(NSAppKitVersionNumber) >= 1671) { /*NSAppKitVersionNumber10_14*/
```

Comment:
Maybe we could keep using the constants for readability, but define them if missing?
```
#ifndef NSAppKitVersionNumber10_12
#define NSAppKitVersionNumber10_12 1504
#endif
#ifndef NSAppKitVersionNumber10_14
#define NSAppKitVersionNumber10_14 1671
#endif
```

Reply:
Yep, looks cleaner this way.